### DISCUSSION TITLE
Enable an English audio recording to be automatically translated

### DISCUSSION THREAD

**Discussion by sysadminpower2019** [ggml-org/whisper.cpp#1921] [Ideas]
Hello,
I noticed that the option to start with a recording in a different language and then automatically transcribe it already exists. Would it be possible to have the opposite? Where the audio starts in English but is instead translated to a different language. Possibly some of the larger languges which are currently well represented in LLMs, German, Spanish, French etc.

**Comment by ulatekh** [comment]
It's my understanding that the existing Whisper models only support two modes, i.e. transcribing English, and translating non-English to English. What you want requires a new model.
